commons-db/src/main/java/com/palantir/nexus/db/sql/AgnosticResultSet.java

```diff
@@ -20,7 +20,7 @@
 
 public interface AgnosticResultSet {
 
-    List<AgnosticResultRow> rows();
+    List<? extends AgnosticResultRow> rows();
```

Comment:
Also as a note changing commons-db should be done with incredible care.  This backs all of the internal projects legacy db connection logic